File: example/src/main/java/thirtytwo/degrees/halfpipe/example/view/IndexController.java
package thirtytwo.degrees.halfpipe.example.view;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

import java.io.IOException;

@Controller
public class IndexController {

    @RequestMapping(value = "/")
    public String home() throws IOException {
        return "forward:/index.html";
    }
}

File: scala-example/src/main/scala/thirtytwo/degrees/halfpipe/scalaexample/view/IndexController.scala
package thirtytwo.degrees.halfpipe.scalaexample.view

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

/**
 * User: spencergibb
 * Date: 9/22/12
 * Time: 4:17 PM
 */
@Controller
class IndexController {

  @RequestMapping(Array("/"))
  def home() = "forward:/index.html"
}
